corehq/apps/case_search/utils.py

```diff
@@ -283,12 +283,17 @@ def get_related_cases(helper, app_id, case_types, cases):
         for _type in types
     ]
 
-    results = []
+    expanded_case_results = []
+    if expand_id_property:
+        expanded_case_results.extend(get_expanded_case_results(helper, expand_id_property, cases))
+
+    results = expanded_case_results
```

Comment:
looks like if the `results` declaration was kept above there wouldn't be a need for `expanded_case_results` and this line-- was that just for clarity?